app/views/announcements/index.html.slim

```diff
@@ -1,5 +1,5 @@
 div.page-header
   h1
     = t('.header')
-= render partial: 'admin/announcements/announcement', collection: @announcements
+= render partial: 'system/admin/announcements/announcement', collection: @announcements
```

Comment:
If so, it should appear in `/views/announcements/_announcement.html.slim` right?